### DISCUSSION TITLE
So I guess the project is dead again ?

### DISCUSSION THREAD

**Discussion by adamency** [aperezdc/termite#11] [Q&A] [answered]
First of all, many many thanks for reviving the project. Moreover you have already implemented the only feature (or rather **fix** on my watch) I was crucially missing which is allowing for protecting against misclicking hyperlinks by adding the need for the Ctrl modifier when clicking, so I'm ever more grateful to you :).

@aperezdc I wanted to make a post to thank you and also in the meantime ask you, given I've seen that the last commits dates from more than 8 months ago now, if the project should be considered dead once again, or at least on hold indefinitely until further notice ?

**Comment by aperezdc** [comment, marked as answer]
@adamency Hi there! I am glad my little side quest to keep Termite working is useful for you. To clear any doubts: the project is **not** dead, it just happened that I'm currently on leave from work (nothing to worry about, FWIW) and right before that I had to focus on leaving a few other things finished. 

I plan to make a new release in the coming days with the fix from #6 and an update to the VTE subproject to its 0.72 branch, but I prefer to include the changes from #9 in a later release down the line.
